pds4indextools/pds4_create_xml_index.py

```diff
@@ -75,42 +107,24 @@ def default_value_for_nil(config, data_type, nil_value):
     return default
 
 
-def convert_header_to_tag(path, root, namespaces):
-    """Convert an XPath expression to an XML tag.
+def grab_elements(xpath):
```

Comment:
grab_elements doesn't really help me understand what this function does. Maybe it's split_into_elements or something like that?